common/src/main/java/com/wynntils/services/map/pois/LabelPoi.java

```diff
@@ -130,13 +130,13 @@ public void renderAt(
                         getTextShadow(),
                         1f);
         if (hovered) {
-            String level = label.getLevel();
-            if (!level.isEmpty()) {
+            int level = label.getLevel();
```

Comment:
... And here we should check that getLevel is not empty first, before checking if it is >= 1.